### ISSUE TITLE
sync has issues

### ISSUE THREAD

**Issue by zwaurykin** [stravo1/obsidian-gdrive-sync#20]
i believe it doesnt sync the text that come particularly at the moment it uploads the file and while syncing again at the refresh rate it deletes that particular text

**Comment by stravo1** [comment]
that is true, when you're making changes while a previous sync is being uploaded, the new sync is added to a sync queue and is uploaded after the previous sync is complete.

if you quit the app before the sync queue is empty then only a part of the entire text is synced and thus in any other device if the same file is opened, it would update it with only the portion that is synced.

however the original file in the original device should not get affected even if there's a partial sync, but i have observed (and other users have also reported) that there are some situations where it does, i will take a look into it.

**Comment by stravo1** [comment]
https://github.com/stravo1/obsidian-gdrive-sync/releases/tag/v0.9.9-beta-8

this should fix the issue, please do test it.

thank you :)
